lib/rmt/downloader.rb

```diff
@@ -68,7 +75,14 @@ def process_queue
     @hydra.queue(request_fiber.resume)
   end
 
+  def deduplicate(checksum_type, checksum_value, destination)
+    return false unless RMT::FileUtils.deduplicate(checksum_type, checksum_value, destination)
+    @logger.info("↓ #{File.basename(destination)}")
```

Comment:
A different symbol would be good. `→` maybe?
SMT has `D` for download and I think `L` for link.